### PR TITLE
Fixes on overview & prolog tutorials

### PR DESCRIPTION
**Pull request by antho31**
* Specify the ichiban predicates list, the difference with SWIPL
* Overview update with the missing link to the CLI tutorial

### CODE REVIEW

**Review by ccamel — APPROVED** [review]

File: docs/tutorials/prolog-1.mdx [hunk 1]
```diff
@@ -395,7 +400,9 @@ height: "2637781"
 '
 ```
 
-## Custom Prolog predicates within the OKP4 blockchain
+## Built-in Prolog predicates within the OKP4 blockchain
+
+The OKP4 blockchain uses the [ichiban/prolog](https://github.com/ichiban/prolog/) Prolog interpreter. Built-in predicates are listed [here](https://github.com/ichiban/prolog/wiki#predicates).
```
Comment (amimart) [inline docs/tutorials/prolog-1.mdx:405]: The logic module interpreter can manage `ichiban/prolog` built-in predicates and our custom predicates, but their availability is conditioned to the interpreter configuration. For instance, the `halt/1` built-in predicate is not registered in the interpreter, its usage allowing to stop a node.

To get the list of available predicates you can query the logic module parameters:

```sh
okp4d --node "https://api.testnet.okp4.network:443/rpc" \
    query params subspace \
    logic Interpreter -ojson \
    | jq -r '.value | fromjson.registered_predicates[]'
```

⚠️ _It's worth noticing that, in the next version the logic module parameters won't be stored in the `params` module anymore, but directly in the logic module itself, so the query to retrieve them will change. And the configuration will be based on white & black lists of predicates_

File: docs/tutorials/prolog-1.mdx [hunk 2]
```diff
@@ -498,7 +505,7 @@ okp4d query logic ask \
 
 #### `consult/1`
 
-The [`consult(:File)`](https://www.swi-prolog.org/pldoc/doc_for?object=consult/1) predicate has been extended to enable data loading from any smart contract query through the `coswasm` prefix.
+The [`consult(:File)`](https://github.com/ichiban/prolog/wiki#program) predicate has been extended to enable data loading from any smart contract query through the `coswasm` prefix.
```
Comment (amimart) [inline docs/tutorials/prolog-1.mdx:508]:
```suggestion
The [`consult(:File)`](https://github.com/ichiban/prolog/wiki#program) predicate has been extended to enable data loading from any smart contract query through the `cosmwasm` URI.
```

**Review by amimart — CHANGES_REQUESTED** [review]
That's great to explain this, I've provided some more elements in remarks to help being more precise.

File: docs/tutorials/prolog-1.mdx (hunk 1 above)
Comment (antho31) [reply docs/tutorials/prolog-1.mdx]: That is a very useful command! Text added, thanks :)

**Review by amimart — APPROVED** [review]

**Review by ccamel — APPROVED** [review]
:+1:

**Review by bdeneux — APPROVED** [review]
Nice ! Thanks 👍

**Pull request merged**